runbot_travis2docker/models/runbot_build.py

```diff
@@ -108,6 +109,11 @@ def job_20_test_all(self, cr, uid, build, lock_path, log_path):
             '--name=' + build.docker_container,
             '-t', build.docker_image,
         ] + pr_cmd_env
+        logdb = cr.dbname
+        if config['db_host'] and grep(build.server('sql_db.py'), 'allow_uri'):
```

Comment:
We don't have the source code of odoo to `grep` method.
`if config['db_host']:` is enough